Find a functionally equivalent code:android.print.PrintManager.getGlobalPrintManagerForUser:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Creates an instance that can access all print jobs.
 *
 * @param userId The user id for which to get all print jobs.
 * @return An instance if the caller has the permission to access all print
 * jobs, null otherwise.
 * @hide
 */

@@ -1,3 +1,7 @@
 {
+    if (mService == null) {
+        Log.w(LOG_TAG, "Feature android.software.print not available");
+        return null;
+    }
     return new PrintManager(mContext, mService, userId, APP_ID_ANY);
 }

